### PR TITLE
navbar sider design

### PR DESCRIPTION
**Pull request by fidan93**
Created NavMenu component. Added Sider with NavMenu inside to all components where NavBar component rendered.

### CODE REVIEW

File: src/components/common/NavBar.js [hunk 1]
```diff
@@ -36,94 +39,49 @@ const StyledNavBar = styled.header`
...
-        onClick={() => history.push('/logout')}
-      >
-        Log Out
-      </Menu.Item>
-    </Menu>
-  );
+    const elem = document.querySelectorAll('.navSider');
+
+    elem.forEach(el => {
+      console.log(el);
```
Comment (eet349) [inline src/components/common/NavBar.js:68]: Can we remove console.logs?

File: src/components/pages/AdminDashboard/AdminDashboardPage.js [hunk 2]
```diff
@@ -43,18 +45,23 @@ function RenderHomePage() {
 
   return (
     <LayoutContainer>
-      <Layout className="layout">
-        <NavBar titleName="Admin Dashboard" />
-        <Layout className="adminDashboardContent">
-          <Sider width={200} className="site-layout-background">
+      {/* <Layout className="layout"> */}
```
Comment (eet349) [inline src/components/pages/AdminDashboard/AdminDashboardPage.js:48]: Can we remove the commented out code?

File: src/components/common/NavBar.js (hunk 1 above)
Comment (fidan93) [reply src/components/common/NavBar.js:68]: yes

File: src/components/common/NavBar.js [hunk 3]
```diff
@@ -36,94 +39,49 @@ const StyledNavBar = styled.header`
...
-      <Menu.Item
-        key="7"
-        icon={<LogoutOutlined />}
-        onClick={() => history.push('/logout')}
-      >
-        Log Out
-      </Menu.Item>
-    </Menu>
-  );
+    const elem = document.querySelectorAll('.navSider');
```
Comment (ashtilawat23) [inline src/components/common/NavBar.js:65]: Nice job cleaning that up,

**Review by ashtilawat23 — APPROVED** [review]

File: src/components/pages/AdminDashboard/AdminDashboardPage.js (hunk 2 above)
Comment (fidan93) [reply src/components/pages/AdminDashboard/AdminDashboardPage.js:48]: yes. It is like old code which was there before me

**Review by eet349 — APPROVED** [review]

**Pull request merged**